gmm/handler.go

```diff
@@ -2320,7 +2370,11 @@ func HandleDeregistrationAccept(ue *context.AmfUe, anType models.AccessType,
 	}
 
 	ue.DeregistrationTargetAccessType = 0
-	return nil
+
+	return GmmFSM.SendEvent(ue.State[models.AccessType__3_GPP_ACCESS], DeregistrationAcceptEvent, fsm.ArgsType{
```

Comment:
why are we sending DeregistrationAcceptEvent? can u please add some comments before the return statement